tests/timeseries/test_quantities.py

```diff
@@ -531,3 +533,26 @@ def test_invalid_multiplications() -> None:
             _ = quantity * 200.0  # type: ignore
         with pytest.raises(TypeError):
             quantity *= 200.0  # type: ignore
+
+
+@pytest.mark.parametrize("quantity_type", [Power, Voltage, Current, Energy, Frequency])
+@pytest.mark.parametrize("exponent", [0, 3, 6, 9])
+@hypothesis.settings(max_examples=1000)
```

Comment:
Any reason to specify a `max_examples`?

Reply:
Mainly for determinism and to limit the total runtime of the test but also to get a decent amount of samples (default value is `100`)